post.hbs

```diff
@@ -144,6 +144,17 @@
     </main>
 {{/post}}
 
-{{!-- Facebook SDK for comments --}}
-<div id="fb-root"></div>
-<script async defer crossorigin="anonymous" src="https://connect.facebook.net/cs_CZ/sdk.js#xfbml=1&version=v3.3"></script>
+{{> "footer"}}
+
+{{#contentFor "styles"}}
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.22.0/themes/prism-okaidia.min.css" />
+{{/contentFor}}
+
+{{#contentFor "scripts"}}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.22.0/prism.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.22.0/plugins/autoloader/prism-autoloader.min.js"></script>
+
+    {{!-- Facebook SDK for comments --}}
+    <div id="fb-root"></div>
+    <script async defer crossorigin="anonymous" src="https://connect.facebook.net/cs_CZ/sdk.js#xfbml=1&version=v3.3"></script>
```

Comment:
Z těchto předgenerovaných kódů od třetích stran jsem pořád zmatená ― proč tam musí být obojí? Nestačil by jen `async` v našem případě?

Reply:
Skript je sice v zápatí, takže ty atributy se nevyužijí úplně naplno, ale přesto aspoň trochu – pokud vím. `async` odblokuje renderování, `defer` odloží vykonání skriptu po načtení a sníží prioritu načítání. Dokonce bych řekl, že je bezpečné takovýhle skript dát do `head`.

Myslím, že ve skriptech by se dalo někdy uklidit, v tom duchu je založená issue #4. Tak či onak bych to nechal, přijde mi to správně.

Reply:
Už se mi podařilo si najít [odpověď](https://www.w3.org/TR/2011/WD-html5-20110525/scripting-1.html#attr-script-async):

> The defer attribute may be specified even if the async attribute is specified, to cause legacy Web browsers that only support defer (and not async) to fall back to the defer behavior instead of the synchronous blocking behavior that is the default.

Určitě bych to teď neřešila, tohle byla taková řečnická otázka, ke které bych se ráda v budoucnu vrátila, klidně až v rámci issue.